flow-tests/test-dev-mode/src/test/java/com/vaadin/flow/uitest/ui/frontend/BrowserLoggingIT.java

```diff
@@ -33,15 +35,30 @@ public class BrowserLoggingIT extends ChromeBrowserTest {
     @Test
     public void nonProductionModeHasLogEntries() {
         open();
-        waitForElementPresent(By.id("elementId"));
+        waitForElementPresent(By.id("exception"));
+
+        findElement(By.id("exception")).click();
+
+        ArrayList<Object> logMessages = (ArrayList<Object>) executeScript(
+                "return window.allLogMessages;");
 
         assertThat(
-                "Flow in production mode should output nothing into the console",
-                getLogEntriesCount(), greaterThan(0L));
-    }
+                "Flow in non-production mode should output something into the console",
```

Comment:
nit: I would use `development mode`, as we usually use that term
```suggestion
                "Flow in development mode should output something into the console",
```